magellan-library/src/main/java/com/wealthfront/magellan/coroutines/CreatedLifecycleScope.kt

```diff
@@ -26,6 +26,6 @@ public class CreatedLifecycleScope @Inject constructor() : LifecycleAware, Corou
   }
 
   override fun destroy(context: Context) {
-    cancel(CancellationException("Destroyed"))
+    job.cancel(CancellationException("Destroyed"))
```

Comment:
possible bug here? I noticed that ShownLifecycleScope is targeting the `job` for cancelation while CreatedLifecycleScope is not

Reply:
Ah good catch! I don't remember off the top of my head which one is "correct", but I think I remember this one being the suggestion.